sn_client/README.md

```diff
@@ -32,6 +33,22 @@ use sn_client::Client;
 let client = Client::new(signer, peers, req_response_timeout, custom_concurrency_limit).await?;
 ```
 
+## Running Tests
+
+Prerequisites:
+* A network running. Could be local. Refer to `safe_network/README.md` to run a local test network.
+* `SAFE_PEERS` environment variable or running the tests with --feature=local-discovery
+
+```bash
+$ cargo test --package sn_client --release --tests --features=local-discovery
+```
+
+## Create docs
+
+```bash
+$ cargo doc --package sn_client
+```
+
```

Comment:
I think we can leave this out. Rust crates will usually have a `docs.rs` icon somewhere in the top of the README.